packages/invoice-template/src/InvoiceTemplate/useInvoiceTemplate.spec.tsx

```diff
@@ -0,0 +1,97 @@
+import { InvoiceItemsTableData, InvoiceItemsTableLabels, LocalizationOptions, TaxesData, TaxesTableLabels } from "..";
+import { useInvoiceTemplate, UseInvoiceTemplateReturn } from "./useInvoiceTemplate";
+
+const netValueString = "€4,000.00";
+const taxValueString = "€1,055.00";
+const totalString = "€5,055.00";
+
+describe("useInvoiceTemplate", () => {
+    let invoiceTemplate: UseInvoiceTemplateReturn;
+
+    beforeAll(() => {
+        invoiceTemplate = useInvoiceTemplate({
+            localizationOptions,
+            invoiceItemsTableData,
+            taxesData,
```

Comment:
Using the same data in all tests can't test too much I think. Should tests cover different scenarios with different aggregations?